### DISCUSSION TITLE
Trying to find with-jest-react-testing-library example

### DISCUSSION THREAD

**Discussion by apardo04** [vercel/next.js#11125] [Help] [answered]
Seems like it once existed from a merge:
https://github.com/zeit/next.js/commit/e9e06099cd4fc70362d44292f4cf32659d782f9a#diff-0a35e670065de6c119696460a4ad7d38

but now I don't see it in the repo anymore.

I see it in this forked repo: https://github.com/Megoos/next.js/tree/update-with-jest/examples/with-jest-react-testing-library

Is there a reason it was removed from next.js repo?

**Comment by Timer** [comment, marked as answer]
All Jest examples were consolidated into a single example:
https://github.com/zeit/next.js/tree/canary/examples/with-jest

**Reply by Timer** [reply]
This example includes `@testing-library/react` usage!

**Reply by apardo04** [reply]
Perfect, I see. thank you so much!
